routes/articles.js

```diff
@@ -0,0 +1,149 @@
+const express = require('express');
+const router = express.Router();
+const articles = require('../db/articles.js');
+
+let error = false;
+let duplicate = false;
+let deleted = false;
+let deletedArticle;
+
+router
+  .route('/')
+  .get((req, res) => {
+    let context = { article: articles.getAllArticles() };
+    if (deleted) {
+      context = { article: articles.getAllArticles() };
+      context.deleteMessage = `You've successfully deleted ${deletedArticle}!!`;
+      deleted = false;
+      deletedArticle = '';
+
+      res.render('layouts/articles/index', context);
```

Comment:
Since you have the same res.render in both your if and your else, you can move it out of the if/else altogether.